homeassistant/components/zwave_js/helpers.py

```diff
@@ -399,6 +397,7 @@ def async_is_device_config_entry_not_loaded(
         raise ValueError(f"Device {device_id} not found")
     return any(
         (entry := hass.config_entries.async_get_entry(entry_id))
+        and entry.domain == DOMAIN
```

Comment:
`async_is_device_config_entry_not_loaded` is not used anymore.

Reply:
Will you remove the function separately?

Reply:
oh whoops thought I removed it in the other PR. OK to remove in here?

Reply:
Yeah, that's ok.